python/pyarrow/tests/test_array.py

```diff
@@ -627,7 +627,7 @@ def test_string_binary_from_buffers():
     assert copied.null_count == 0
 
 
-@pytest.mark.parametrize('list_type_factory', [pa.list_, pa.large_list])
+@pytest.mark.parametrize('list_type_factory', [pa.list_, pa.large_list, pa.list_view, pa.large_list_view])
 def test_list_from_buffers(list_type_factory):
```

Comment:
I would expect that this test needs some slight changes to work for the list view types? Because those use 3 instead of 2 buffers for the main array (apart from the child values)?